modules/rosa-policy-change-management.adoc

```diff
@@ -62,6 +62,4 @@ OpenShift Container Platform software and the underlying immutable Red Hat CoreO
 [id="rosa-policy-release-management_{context}"]
 == Release management
 
-ROSA clusters can be configured for automatic upgrades on a schedule. Alternatively, you can perform manual upgrades using the `rosa` CLI. For more details, see the link:https://access.redhat.com/support/policy/updates/openshift/dedicated[Life Cycle policy].
-
-Customers can review the history of all cluster upgrade events in their {cluster-manager} web console on the Events tab.
+Red Hat does not automatically upgrade your clusters. You can schedule to upgrade the clusters at regular intervals (automatic upgrade) or just once (manual upgrade) using the {cluster-manager} web console. Red Hat might forcefully upgrade a cluster to a new z-stream version only if the cluster is affected by a critical impact CVE. You can review the history of all cluster upgrade events in the {cluster-manager} web console. For more information about releases, see the link:https://access.redhat.com/support/policy/updates/openshift/dedicated[Life Cycle policy].
```

Comment:
(automatic/recurring upgrade) or just once (manual/scheduled upgrade) using the {cluster-manager} web console or ROSA CLI.

Reply:
@sayjadha We also need to point out that ROSA STS clusters always need an interruption between upgrades, due to needing to ensure STS roles/policies are up to date.

Reply:
Hi @arendej : Updated the doc per your feedback. Please review and confirm the changes? Thank you.